.env.example

```diff
@@ -14,6 +14,7 @@ VUE_APP_SITEMINDER_LOGOUT_URL="https://logontest7.gov.bc.ca/clp-cgi/logoff.cgi"
 VUE_APP_BUSINESS_CREATE_URL="https://dev.create.business.bcregistry.gov.bc.ca/"
 VUE_APP_BUSINESS_EDIT_URL="https://dev.edit.business.bcregistry.gov.bc.ca/"
 VUE_APP_BUSINESS_DASH_URL="https://dev.business-dashboard.bcregistry.gov.bc.ca/"
+VUE_APP_BUSINESS_REG_DASH_URL="https://dev.business-registry-dashboard.bcregistry.gov.bc.ca/"
```

Comment:
You also need to update https://github.com/bcgov/business-filings-ui/blob/main/devops/vaults.env so that the build/deploy process knows where to get this value from 1Password.

Hint:
- vault is `web-url`
- section is `business-dash`
- key is `BUSINESS_DASH_URL`

Reply:
Fixed, thanks.